perception/autoware_image_projection_based_fusion/src/utils/utils.cpp

```diff
@@ -14,8 +14,31 @@
 
 #include "autoware/image_projection_based_fusion/utils/utils.hpp"
 
+#include <sensor_msgs/distortion_models.hpp>
+
 namespace autoware::image_projection_based_fusion
 {
+bool checkCameraInfo(const sensor_msgs::msg::CameraInfo & camera_info)
+{
+  bool is_supported_model =
+    (camera_info.distortion_model == sensor_msgs::distortion_models::PLUMB_BOB ||
+     camera_info.distortion_model == sensor_msgs::distortion_models::RATIONAL_POLYNOMIAL);
+  if (!is_supported_model) {
+    RCLCPP_WARN_STREAM(
```

Comment:
I think it is OK to make it an error instead of a warning

Reply:
The purpose of this PR is to make it alive even the input is wrong.
In some reason, probability of the error reproduction became very low. 
Let's make it to 'ERROR' so that we can catch a hint.

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/7747/commits/b96c90270e15bd4d8e43d0114aaf7274eb38cdb9